File: delivery/DeliveryReturned.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class DeliveryReturned(AbstractEvent):
    id : int
    orderId : str
    productId : str
    productName : str
    qty : int
    amount : int
    deliveryDt : 
    status : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.orderId = None
        self.productId = None
        self.productName = None
        self.qty = None
        self.amount = None
        self.deliveryDt = None
        self.status = None

File: customercenter/src/main/java/shopmyid/domain/DeliveryReturned.java
package shopmyid.domain;

import java.time.LocalDate;
import java.util.*;
import lombok.Data;
import shopmyid.infra.AbstractEvent;

@Data
public class DeliveryReturned extends AbstractEvent {

    private Long id;
    private String orderId;
    private String productId;
    private String productName;
    private Integer qty;
    private Integer amount;
    private Date deliveryDt;
    private String status;
}
